sdk/core/core-lro/src/http/operation.ts

```diff
@@ -180,7 +180,7 @@ export function parseRetryAfter<T>({ rawResponse }: LroResponse<T>): number | un
 }
 
 export function getErrorFromResponse<T>(response: LroResponse<T>): LroError | undefined {
-  const error = (response.flatResponse as ResponseBody).error;
+  const error = accessBodyProperty(response, "error");
```

Comment:
Interesting it seems RLC didn't realize this and no testing covered.

Reply:
If the LRO errors, the error is extracted from the response to be added to the poller's state.

This change makes sure to look for the error in both the parsed response and the raw response.